cmd/root.go

```diff
@@ -11,6 +11,23 @@ import (
 	"github.com/elastic/elastic-package/internal/logger"
 )
 
+// Make sure to preserve alphabetical order!
```

Comment:
nit: unless you introduce dedicated array `type Commands []Command` that has a method `sorted()` :)

Reply:
Yeah, I had done this before but decided to remove it and keep it simple since we control the addition of commands (it's not a truly open plugin-like system). So I'm going to keep it this way for now.

Reply:
Actually, I added sorting in 55c1edd.